### ISSUE TITLE
JSONL support?

### ISSUE THREAD

**Issue by billcrook** [leonchen83/redis-rdb-cli#13]
**Ask a question**
Any possibility of getting [JSONL](http://jsonlines.org/) export support? This is a common format used in big data tooling. [JsonRdbVisitor.java](https://github.com/leonchen83/redis-rdb-cli/blob/master/src/main/java/com/moilioncircle/redis/rdb/cli/ext/rct/JsonRdbVisitor.java) could be modified to take a new constructor arg `jsonl: Boolean`. When enabled, it could be used to omit the wrapping array characters and use newlines instead of commas to separate the entries.

Thanks again for this great tool!

**Comment by leonchen83** [comment]
@billcrook 
yes, that is a great idea.  
but we need to choose some of format.  
in json export, the format is like following  
```
[
    {db0`s kv1, kv2, kv3, kv4}, 
    {db1's kv1, kv2, kv3, kv4},
    ....
]
```

there are 2 format to choose in jsonl export
one is ignore db's information and expiry like following 
```
{"k":"v"}\n
{"k1":"v1"}\n
{"k2":"v2"}\n
...
```

another one have the db information and its expiry like following  

```
{db:0, "key": "k", "value": "v", “expiry”:1583462335381}\n
{db:1, "key": "k1", "value": "v1", “expiry”:1583462335381}\n
{db:2, "key": "k2", "value": "v2", “expiry”:1583462335381}\n
```

second choice would be best, what do you think?

**Comment by billcrook** [comment]
I would actually prefer the first choice, but it is likely due to my use case. I am using the keys regex, so I don't need the type or db information since I know the type of the keys and the db they are coming from. In this case, the extra meta information would just be repeated and increase the jsonl file size.
 
Maybe we could we add a flag to the cli? Something like `include_meta`? True would cause the export to include db, type and expiry. False would not.

Any of these options would ultimately work for me (first or second approach, with or without the flag).  Obviously your call, since this is your project :)

Thanks again for the great tool!

**Comment by leonchen83** [comment]
@billcrook 
`include_meta ` is a good idea.
I will add an extra parameter `export_meta=true` in `/path/to/redis-rdb-cli/conf/redis-rdb-cli.conf`
by default this value is true means the exported data like following
```
{"db":0, "key": "k", "value": "v", "expiry": 1583462335381, "type": "string"},
{"db":1, "key": "k1", "value": "v1", "expiry": 1583462335381, "type":"zset"},
{"db":2, "key": "k2", "value": "v2", "expiry": 1583462335381, "type": "hash"},
{"db":2, "key": "k3", "value": "v3", "type": "string"},
```
and if this value is false. the data will be
```
{"key": "k", "value": "v"},
{"key": "k1", "value": "v1"},
{"key": "k2", "value": "v2"},
{"key": "k3", "value": "v3"},
```

**Comment by billcrook** [comment]
Perfect!

**Comment by leonchen83** [comment]
@billcrook 
fixed at version [v0.4.0](https://github.com/leonchen83/redis-rdb-cli/releases/download/v0.4.0/redis-rdb-cli-release.zip)

**Comment by billcrook** [comment]
Test successful! I tested both values of `export_meta` and it's looking good.

You are the man! Thanks again
